skrf/media/media.py

```diff
@@ -142,7 +141,8 @@ def z0(self):
     def z0(self, val):
         self._z0 = val
 
-    @abstractproperty
+    @property
+    @abstractmethod
```

Comment:
i am not sure the difference between these two.  are they equivalent?

Reply:
According to python docs it's equivalent, and recommended since python3.3.
I think there should be another cleanup round to remove lot of python2 related code. I just came over this recently, so I fixed this directly.